app/cdash/app/Model/Build.php

```diff
@@ -623,9 +617,8 @@ public function GetResolvedBuildErrors($type)
      *
      * @param int $fetchStyle
      * @param array $filters
-     * @return array|bool
      */
-    public function GetErrors(array $propertyFilters = [], $fetchStyle = PDO::FETCH_ASSOC)
+    public function GetErrors(array $propertyFilters = [], int $fetchStyle = PDO::FETCH_ASSOC) : array|false
```

Comment:
The PHPDoc comments above this function are wrong (and unnecessary since there are inline type declarations) and should be removed.